Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -171,19 +171,43 @@ void GCPad::LoadDefaults(const ControllerInterface& ciface)
   EmulatedController::LoadDefaults(ciface);
 
   // Buttons
-  m_buttons->SetControlExpression(0, "`X`");  // A
-  m_buttons->SetControlExpression(1, "`Z`");  // B
-  m_buttons->SetControlExpression(2, "`C`");  // X
-  m_buttons->SetControlExpression(3, "`S`");  // Y
-  m_buttons->SetControlExpression(4, "`D`");  // Z
+#if _UWP
```

Comment:
We can check at runtime if this is an Xbox device, though this might not be needed. All UWP supported Xbox models see the controller as 'Xbox One Game Controller', regardless of if the app is running on a Series S/X or One (X).